components/popover/popover-mixin.js

```diff
@@ -125,25 +153,42 @@ export const PopoverMixin = superclass => class extends superclass {
 		this._dismissibleId = null;
 	}
 
-	_close() {
-		const hide = () => {
-			this.opened = false;
-		};
+	_focusContent(container) {
```

Comment:
This method accepts a target element to query because eventually, `PopoverMixin` will be updated to support a focus-trap in the same way as dropdown. In such a case, `container` will be an element underneath the focus-trap, rather than `this`. So... just setting things up.